src/helpers/daily-data-providers/combined-resting-heart-rate.ts

```diff
@@ -1,5 +1,10 @@
 ﻿import { add } from "date-fns";
-import { appleHealthRestingHeartRateDataProvider, fitbitRestingHeartRateDataProvider, garminRestingHeartRateDataProvider } from ".";
+import {
```

Comment:
We'll need to update the availability check for RHR/sleep combined as well: https://github.com/CareEvolution/MyDataHelpsUI/blob/main/src/helpers/daily-data-types/combined.tsx#L17

Reply:
Which should also incorporate getDeviceDataV2AllDataTypes()